stdlib/builtins.pyi

```diff
@@ -1104,7 +1104,7 @@ class list(MutableSequence[_T]):
     def __init__(self) -> None: ...
     @overload
     def __init__(self, iterable: Iterable[_T], /) -> None: ...
-    def copy(self) -> list[_T]: ...
+    def copy(self) -> list[_S | _T]: ...
```

Comment:
`_S` has no meaning, as it's only used once in this context. It's just interpreted as `Any` in this context.

Reply:
I see what you're trying to do, but we should just use:

```suggestion
    # The new list can allow broader types that the old one. `Any` allows that.
    def copy(self) -> list[_T | Any]: ...
```

Reply:
consider this example:
```py
def empty_list[T]() -> list[T]:
    return []

a = empty_list()
b: list[int] = empty_list()
```
if we were to use `Any` here, then the type of `a` would be `list[Any]`

additionally, there would be absolutely no safety within `empty_list` to return the correct type. i'm not really sure where this idea that type variables only in return positions came from

this is literally the meaning of a type variable, to use `Any` is a mistake

Reply:
> basedpyright is incorrect there. `A[int | Any]` should be assignable to `A[object]`.

yes: https://github.com/microsoft/pyright/issues/10713#issuecomment-3132207928

i was a little hasty with the first example, i think it's clear though that this case is the intended use of a type variable, not `Any`